### PR TITLE
Update participants.json

### PR DESCRIPTION
**Pull request by kavi-12**

### CODE REVIEW

File: data/participants.json [hunk 1]
```diff
@@ -1649,6 +1649,16 @@
 			"twitter"   : "",
 			"referral"  : ""
 
+		},
+	    	{
+			"name"      : "VIDUR KATARIA",
+			"imageur1"  : "assets/img/participants/DSC_8595.jpg",
```
Comment (DreamyPhobic) [inline data/participants.json]: Also add image. Currently,there is no image with this name. And image name should be your username or firstname ...

File: data/participants.json [hunk 2]
```diff
@@ -1652,7 +1652,7 @@
 		},
 	    	{
 			"name"      : "VIDUR KATARIA",
-			"imageur1"  : "assets/img/participants/DSC_8595.jpg",
+			"imageur1"  : "assets/img/participants/vidur.jpg",
```
Comment (DreamyPhobic) [inline data/participants.json]: where is your image??

**Comment by kavi-12** [comment]
sir  i have uploaded it just know

**Comment by Tlazypanda** [comment]
@kavi-12 The merge conflicts have been resolved by us.Kindly upload your image so that we can proceed with merging.

**Pull request merged**